tests/zfs-tests/tests/functional/quota/quota.kshlib

```diff
@@ -62,7 +62,7 @@ function fill_quota
 
 	typeset -i file_size=`$LS -ls $MNTPT/$TESTFILE1 | $AWK '{ print $1 }'`
 	typeset -i limit=0
-	(( file_size = file_size * 512 ))
+	(( file_size = file_size * $BLOCK_SIZE ))
```

Comment:
This should be 1024not $BLOCK_SIZE.  Under Linux the default `ls` output is in 1k blocks instead of 512b blocks.  This could be added to the existing `is_linux` block above.  See https://www.gnu.org/software/coreutils/manual/html_node/Block-size.html for complete details.